integration-tests/tests/resource-observer/resources.ejs

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Resource observer test</title>
+  <%- renderAgent() %>
+</head>
+<body>
+  <h1>Resource observer test</h1>
+
+  <button id="load-img">Load new Image</button>
+  <button id="load-script">Load new script</button>
+
+  <img id="img-1" src="/integration-tests/assets/splunk-black-white.png" alt="black">
```

Comment:
Can you move https://github.com/signalfx/splunk-browser-otel-js/blob/main/performance-tests/devServer/randomImageProvider.js to test-utils, and mount in the integration tests server instead of these static images? It might get a bit more tricky if we wanted to assert content length, but could be solved with hard-coded seed.

Reply:
What's wrong with static images? Every test that needs an image can just use this one. Pretty much need one of each asset (css, img, .js) for all of the tests. Actually this image tag is even unneeded here I'll just remove it.